shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -470,6 +470,28 @@ object JsonLdUtils {
                 else it.toUri().right()
             }
     }
+
+    fun constructJsonLdProperty(value: Any): MutableMap<String, Any> =
+        mutableMapOf(
+            JSONLD_TYPE to NGSILD_PROPERTY_TYPE.uri,
+            NGSILD_PROPERTY_VALUE to mapOf(
+                JSONLD_VALUE_KW to value
+            )
+        )
+
+    fun constructJsonLdRelationship(value: Any): MutableMap<String, Any> =
+        mutableMapOf(
+            JSONLD_TYPE to NGSILD_RELATIONSHIP_TYPE.uri,
+            NGSILD_RELATIONSHIP_HAS_OBJECT to mapOf(
+                JSONLD_ID to value
+            )
+        )
+
+    fun constructJsonLdDateTime(value: Any): Map<String, Any> =
+        mapOf(
+            JSONLD_TYPE to NGSILD_DATE_TIME_TYPE,
+            JSONLD_VALUE_KW to value
+        )
```

Comment:
Good move.

A few points:
- the return types could all be Map<String, Any> (instead of returning a mutable map)
- use the most specific types for the methods arguments (it makes the contract clear and reduces the risk of errors). for instance, for a relationship, you are expecting a `URI` type, ...
- use the prefix `build` instead of `construct` (more common and in line with current code). also, will be clearer if you name them like `buildJsonLdExpandedRelationship` (always good info to know if we are working with an expanded or compacted form)